nservicebus/azure-service-bus/send-atomic-with-receive-note.include.md

```diff
@@ -1 +1 @@
-Note: `TransportTransactionMode.SendsAtomicWithReceive` level is only supported in Versions 7 and above of the Azure Service Bus transport. It requires destination and receive queues to be in the same namespace.
\ No newline at end of file
+Note: `TransportTransactionMode.SendsAtomicWithReceive` level is only supported in Versions 7 and above of the Azure Service Bus transport. It requires destination and receive queues to be in the same namespace and the use of the NetMessaging transport type.
```

Comment:
>Note: `TransportTransactionMode.SendsAtomicWithReceive` level is only supported in Versions 7 and above of the Azure Service Bus transport. It requires destination and receive queues to be in the same namespace and the use of the NetMessaging transport type.

How about:

INFO: `TransportTransactionMode.SendsAtomicWithReceive` level is only supported in Versions 7 and above of the Azure Service Bus transport. It requires both destination and receive queues to be in the same namespace. Also, the transport type must be set to `TransportType.NetMessaging` instead of `TransportType.Amqp`.